package.json

```diff
@@ -1,18 +1,30 @@
 {
   "name": "btcpay",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "A nodejs client implementation for BTCPay",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc"
   },
-  "author": "Tim Akinbo <tim@tanjalo.com>",
+  "author": "Tim Akinbo <tim@tanjalo.com>; Christoph Ott <christoph.ott@lean-coders.at>",
   "license": "MIT",
   "dependencies": {
+    "@types/bs58": "^4.0.0",
```

Comment:
I thought types dependencies were dev time dependencies (Node noob here, am I wrong?)

Reply:
we could move it - didn't see it as necessary. If you like you can change it. https://stackoverflow.com/questions/45176661/how-do-i-decide-whether-types-goes-into-dependencies-or-devdependencies